packages/admin/src/views/Admin.js

```diff
@@ -6,13 +6,48 @@
  *
  */
 
-import { useNavigate } from 'react-router-dom'
-import { useServices, Navbar, FooterSmall } from '@kernel/common'
+import { useNavigate, Link } from "react-router-dom"
+import { useServices, Navbar, FooterSmall } from "@kernel/common"
 
-import bgImage from 'assets/images/admin_bg.png'
+import bgImage from "assets/images/admin_bg.png"
 
 const ADMIN_ROLE = 100 
 
+const navbarLinks = [
```

Comment:
I think this is a good time to introduce a `App.config.json` configuration for our apps to put all these `const` I have been using across all the different files.

Let's put it next to our `App.js` @ `packages/admin/src/App.config.json`:
```
{
  "appTitle": "Kernel Steward",
  "adminRole": 100,
  "navbarLinks": [ ... ],
  ...
}
```

We can then just import as -
```
import AppConfig from 'AppConfig'
```

and use it like

```
<Navbar title={ AppConfig.appTitle } ... />
```
Let's start with that approach and just import it at the top of the file. 

As a next step, we may move to using a `Context` [0] especially once we want to support things like dark mode and other variations. Alternatively, we could also pass it down the components tree from the root. Pros and cons to both approaches for us to consider.

[0] https://reactjs.org/docs/context.html